ioread.cpp

```diff
@@ -1221,9 +1298,11 @@ bool exec_readGrid(dccrg::Dccrg<SpatialCell,dccrg::Cartesian_Geometry>& mpiGrid,
    phiprof::start("readFsGrid");
    // Read fsgrid data back in
    int fsgridInputRanks=0;
+   phiprof::start("readScalarParameter");
    if(readScalarParameter(file,"numWritingRanks",fsgridInputRanks, MASTER_RANK, MPI_COMM_WORLD) == false) {
       exitOnError(false, "(RESTART) FSGrid writing rank number not found in restart file", MPI_COMM_WORLD);
```

Comment:
Early return doesn't stop timers. Recommended to change all timers to objects i.e. `phiprof::start(label)` -> `phiprof::Timer foo {label}` and phiprof::stop(label, args)` -> `foo.stop(args)`.

Reply:
But wait this guy I inserted and I can make it an object, but the other guys before and after this are also old-style. Why aren't they changed?

Reply:
And there's early exit ones there too.

Reply:
Did you merge dev first? Ah, I suppose some might've slipped in between my updates to timers and merging that to dev...